convert_to_tflite.py

```diff
@@ -0,0 +1,30 @@
+import tensorflow as tf
```

Comment:
It's starting to look like a CLI. In case you want to try one, you can look at https://github.com/google/python-fire

Reply:
thanks for the suggestion, but I am going to use "absl"